CDCgov/prime-reportstream
# Settings API - Use all authentication options

This PR updates the Settings API to both use server2server auth in addition to Okta auth.

Test Steps:
* Run the smoke test
* Verify the code makes sense

## Changes
- Updated `SettingsFunctions` to use both Okta and Server2Server during the authentication process
- Made all tests using copied Okta auth functions pull from a new location, with a new name to stop a function name conflict

## Linked Issues
- #1946
